packages/genesys-spark-components/src/components/stable/gux-tabs/gux-tabs.tsx

```diff
@@ -103,8 +103,8 @@ export class GuxTabs {
       this.activeTab = panels[0].tabId;
     }
 
-    void tabList.guxSetActive(this.activeTab);
-    panels.forEach(
+    void tabList?.guxSetActive(this.activeTab);
```

Comment:
I notice `tabList` gets set in `onSlotchange`. Do we want to log a warning there if `tabListSlot.assignedElements()[0]` is not defined?